include/ignition/gazebo/rendering/SceneManager.hh

```diff
@@ -149,13 +149,20 @@ inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
     public: rendering::LightPtr CreateLight(Entity _id,
         const sdf::Light &_light, Entity _parentId);
 
-    /// \brief Create a particle emitter
+    /// \brief Create a default particle emitter
     /// \param[in] _id Unique particle emitter id
     /// \param[in] _emitter Particle emitter data
     /// \param[in] _parentId Parent id
-    /// \return Particle emitter object created
-    public: rendering::ParticleEmitterPtr CreateParticleEmitter(Entity _id,
-        const msgs::ParticleEmitter &_emitter, Entity _parentId);
+    /// \return Default particle emitter object created
+    public: rendering::ParticleEmitterPtr CreateDefaultParticleEmitter(
+        Entity _id, const msgs::ParticleEmitter &_emitter, Entity _parentId);
+
+    /// \brief Update an existing particle emitter
+    /// \brief _id Emitter id to update
+    /// \brief _emitter Data to update the particle emitter
+    /// \return Particle emitter updated
+    public: rendering::ParticleEmitterPtr UpdateParticleEmitter(Entity _id,
```

Comment:
Minor comment, but I wonder if re-naming this funtion to `ConfigureParticleEmitter` would be clearer.

Reply:
I think the convention for using Configure in ign-gazebo is generally when some logic is executed called only once on startup but here it's updating the params after launch and can be done multiple times so it's using Update instead.